cms/static/js/spec/views/active_video_upload_list_spec.js

```diff
@@ -89,6 +91,40 @@ define(
                 });
             });
 
+            describe('file size upload', function() {
+                it('blocks file uploads larger than the max file size', function() {
```

Comment:
Can we write a single test in place of these 3 tests, that test can check all these scenarios.